core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -79,7 +79,7 @@ trait TestDriverComponent extends DriverComponent with DataAccess with DefaultIn
   // TODO: can we be any more targeted about inTransaction vs. inTransactionWithAttrTempTable here?
   // this is used by many tests to set up fixture data, which includes saving entities, therefore it needs
   // the temp table.
-  protected def runAndWait[R](action: DBIOAction[R, _ <: NoStream, _ <: Effect], duration: Duration = 1 minutes): R = {
+  def runAndWait[R](action: DBIOAction[R, _ <: NoStream, _ <: Effect], duration: Duration = 1 minutes): R = {
```

Comment:
why is this not protected anymore?

Reply:
Because I need to use it in a test outside of the inheritance hierachy (hence my plea to stop using inheritance)

Reply:
Can't say I've heard this plea, where can I hear more?